### ISSUE TITLE
Pending for many hours passed, never enter next step.  No error but why stuck?

### ISSUE THREAD

**Issue by kanglue** [cocktailpeanut/fluxgym#336]
[2025-02-15 17:14:50] [INFO] Running E:\ai\github\fluxgym\outputs\alysa-lora\train.bat
[2025-02-15 17:14:50] [INFO] 
[2025-02-15 17:14:50] [INFO] E:\ai\github\fluxgym>accelerate launch   --mixed_precision bf16   --num_cpu_threads_per_process 1   sd-scripts/flux_train_network.py   --pretrained_model_name_or_path "E:\ai\github\fluxgym\models\unet\flux1-dev.sft"   --clip_l "E:\ai\github\fluxgym\models\clip\clip_l.safetensors"   --t5xxl "E:\ai\github\fluxgym\models\clip\t5xxl_fp16.safetensors"   --ae "E:\ai\github\fluxgym\models\vae\ae.sft"   --cache_latents_to_disk   --save_model_as safetensors   --sdpa --persistent_data_loader_workers   --max_data_loader_n_workers 2   --seed 42   --gradient_checkpointing   --mixed_precision bf16   --save_precision bf16   --network_module networks.lora_flux   --network_dim 4   --optimizer_type adafactor   --optimizer_args "relative_step=False" "scale_parameter=False" "warmup_init=False"   --split_mode   --network_args "train_blocks=single"   --lr_scheduler constant_with_warmup   --max_grad_norm 0.0   --learning_rate 8e-4   --cache_text_encoder_outputs   --cache_text_encoder_outputs_to_disk   --fp8_base   --highvram   --max_train_epochs 16   --save_every_n_epochs 4   --dataset_config "E:\ai\github\fluxgym\outputs\alysa-lora\dataset.toml"   --output_dir "E:\ai\github\fluxgym\outputs\alysa-lora"   --output_name alysa-lora   --timestep_sampling shift   --discrete_flow_shift 3.1582   --model_prediction_type raw   --guidance_scale 1   --loss_type l2
[2025-02-15 17:14:58] [INFO] The following values were not passed to `accelerate launch` and had defaults used instead:
[2025-02-15 17:14:58] [INFO] `--num_processes` was set to a value of `0`
[2025-02-15 17:14:58] [INFO] `--num_machines` was set to a value of `1`
[2025-02-15 17:14:58] [INFO] `--dynamo_backend` was set to a value of `'no'`
[2025-02-15 17:14:58] [INFO] To avoid this warning pass in values for each of the problematic parameters or run `accelerate config`.
[2025-02-15 17:15:01] [INFO] The installed version of bitsandbytes was compiled without GPU support. 8-bit optimizers, 8-bit multiplication, and GPU quantization are unavailable.
[2025-02-15 17:15:03] [INFO] 2025-02-15 17:15:03 INFO     highvram is enabled /           train_util.py:4296
[2025-02-15 17:15:03] [INFO] highvramが有効です
[2025-02-15 17:15:03] [INFO] WARNING  cache_latents_to_disk is        train_util.py:4313
[2025-02-15 17:15:03] [INFO] enabled, so cache_latents is
[2025-02-15 17:15:03] [INFO] also enabled /
[2025-02-15 17:15:03] [INFO] cache_latents_to_diskが有効なた
[2025-02-15 17:15:03] [INFO] め、cache_latentsを有効にします
[2025-02-15 17:15:03] [INFO] 2025-02-15 17:15:03 INFO     Checking the state dict:          flux_utils.py:43
[2025-02-15 17:15:03] [INFO] Diffusers or BFL, dev or schnell
[2025-02-15 17:15:04] [INFO] 2025-02-15 17:15:04 INFO     t5xxl_max_token_length:  flux_train_network.py:152
[2025-02-15 17:15:04] [INFO] 512
[2025-02-15 17:15:05] [INFO] E:\AI\miniconda3\envs\aigc\Lib\site-packages\transformers\tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884
[2025-02-15 17:15:05] [INFO] warnings.warn(
[2025-02-15 17:15:05] [INFO] You are using the default legacy behaviour of the <class 'transformers.models.t5.tokenization_t5.T5Tokenizer'>. This is expected, and simply means that the `legacy` (previous) behavior will be used so nothing changes for you. If you want to use the new behaviour, set `legacy=False`. This should only be set if you understand what it means, and thoroughly read the reason why this was added as explained in https://github.com/huggingface/transformers/pull/24565
[2025-02-15 17:15:05] [INFO] 2025-02-15 17:15:05 INFO     Loading dataset config from   train_network.py:446
[2025-02-15 17:15:05] [INFO] E:\ai\github\fluxgym\outputs\
[2025-02-15 17:15:05] [INFO] alysa-lora\dataset.toml
[2025-02-15 17:15:05] [INFO] INFO     prepare images.                 train_util.py:2053
[2025-02-15 17:15:05] [INFO] INFO     get image size from name of     train_util.py:1944
[2025-02-15 17:15:05] [INFO] cache files
[2025-02-15 17:15:05] [INFO] 0%|          | 0/30 [00:00<?, ?it/s]
100%|██████████| 30/30 [00:00<00:00, 30023.65it/s]
[2025-02-15 17:15:05] [INFO] INFO     set image size from cache       train_util.py:1974
[2025-02-15 17:15:05] [INFO] files: 0/30
[2025-02-15 17:15:05] [INFO] INFO     found directory                 train_util.py:2000
[2025-02-15 17:15:05] [INFO] E:\ai\github\fluxgym\datasets\a
[2025-02-15 17:15:05] [INFO] lysa-lora contains 30 image
[2025-02-15 17:15:05] [INFO] files
[2025-02-15 17:15:05] [INFO] read caption:   0%|          | 0/30 [00:00<?, ?it/s]
read caption: 100%|██████████| 30/30 [00:00<00:00, 5997.29it/s]
[2025-02-15 17:15:05] [INFO] INFO     300 train images with repeats.  train_util.py:2096
[2025-02-15 17:15:05] [INFO] INFO     0 reg images with repeats.      train_util.py:2100
[2025-02-15 17:15:05] [INFO] WARNING  no regularization images /      train_util.py:2105
[2025-02-15 17:15:05] [INFO] 正則化画像が見つかりませんでし
[2025-02-15 17:15:05] [INFO] た
[2025-02-15 17:15:05] [INFO] INFO     [Dataset 0]                     config_util.py:575
[2025-02-15 17:15:05] [INFO] batch_size: 1
[2025-02-15 17:15:05] [INFO] resolution: (512, 512)
[2025-02-15 17:15:05] [INFO] enable_bucket: False
[2025-02-15 17:15:05] [INFO] 
[2025-02-15 17:15:05] [INFO] [Subset 0 of Dataset 0]
[2025-02-15 17:15:05] [INFO] image_dir:
[2025-02-15 17:15:05] [INFO] "E:\ai\github\fluxgym\datasets\
[2025-02-15 17:15:05] [INFO] alysa-lora"
[2025-02-15 17:15:05] [INFO] image_count: 30
[2025-02-15 17:15:05] [INFO] num_repeats: 10
[2025-02-15 17:15:05] [INFO] shuffle_caption: False
[2025-02-15 17:15:05] [INFO] keep_tokens: 1
[2025-02-15 17:15:05] [INFO] caption_dropout_rate: 0.0
[2025-02-15 17:15:05] [INFO] caption_dropout_every_n_epo
[2025-02-15 17:15:05] [INFO] chs: 0
[2025-02-15 17:15:05] [INFO] caption_tag_dropout_rate:
[2025-02-15 17:15:05] [INFO] 0.0
[2025-02-15 17:15:05] [INFO] caption_prefix: None
[2025-02-15 17:15:05] [INFO] caption_suffix: None
[2025-02-15 17:15:05] [INFO] color_aug: False
[2025-02-15 17:15:05] [INFO] flip_aug: False
[2025-02-15 17:15:05] [INFO] face_crop_aug_range: None
[2025-02-15 17:15:05] [INFO] random_crop: False
[2025-02-15 17:15:05] [INFO] token_warmup_min: 1,
[2025-02-15 17:15:05] [INFO] token_warmup_step: 0,
[2025-02-15 17:15:05] [INFO] alpha_mask: False
[2025-02-15 17:15:05] [INFO] custom_attributes: {}
[2025-02-15 17:15:05] [INFO] is_reg: False
[2025-02-15 17:15:05] [INFO] class_tokens: alysa
[2025-02-15 17:15:05] [INFO] caption_extension: .txt
[2025-02-15 17:15:05] [INFO] 
[2025-02-15 17:15:05] [INFO] 
[2025-02-15 17:15:05] [INFO] INFO     [Prepare dataset 0]             config_util.py:587
[2025-02-15 17:15:05] [INFO] INFO     loading image sizes.             train_util.py:972
[2025-02-15 17:15:05] [INFO] 0%|          | 0/30 [00:00<?, ?it/s]
100%|██████████| 30/30 [00:00<00:00, 14985.01it/s]
[2025-02-15 17:15:05] [INFO] INFO     prepare dataset                  train_util.py:997
[2025-02-15 17:15:05] [INFO] WARNING  split_mode is deprecated. flux_train_network.py:76
[2025-02-15 17:15:05] [INFO] Please use
[2025-02-15 17:15:05] [INFO] `--blocks_to_swap`
[2025-02-15 17:15:05] [INFO] instead.
[2025-02-15 17:15:05] [INFO] `--blocks_to_swap 18` is
[2025-02-15 17:15:05] [INFO] automatically set. /
[2025-02-15 17:15:05] [INFO] split_modeは非推奨です。
[2025-02-15 17:15:05] [INFO] 代わりに`--blocks_to_swap
[2025-02-15 17:15:05] [INFO] `を使用してください。`--b
[2025-02-15 17:15:05] [INFO] locks_to_swap
[2025-02-15 17:15:05] [INFO] 18`が自動的に設定されまし
[2025-02-15 17:15:05] [INFO] た。
[2025-02-15 17:15:05] [INFO] INFO     preparing accelerator         train_network.py:520
[2025-02-15 17:15:05] [INFO] accelerator device: cpu
[2025-02-15 17:15:05] [INFO] INFO     Checking the state dict:          flux_utils.py:43
[2025-02-15 17:15:05] [INFO] Diffusers or BFL, dev or schnell
[2025-02-15 17:15:05] [INFO] INFO     Building Flux model dev from BFL flux_utils.py:101
[2025-02-15 17:15:05] [INFO] checkpoint
[2025-02-15 17:15:05] [INFO] INFO     Loading state dict from          flux_utils.py:118
[2025-02-15 17:15:05] [INFO] E:\ai\github\fluxgym\models\unet
[2025-02-15 17:15:05] [INFO] \flux1-dev.sft
[2025-02-15 17:15:06] [INFO] 2025-02-15 17:15:06 INFO     Loaded Flux: <All keys matched   flux_utils.py:137
[2025-02-15 17:15:06] [INFO] successfully>
[2025-02-15 17:15:06] [INFO] INFO     Cast FLUX model to fp8.  flux_train_network.py:103
[2025-02-15 17:15:06] [INFO] This may take a while.
[2025-02-15 17:15:06] [INFO] You can reduce the time
[2025-02-15 17:15:06] [INFO] by using fp8 checkpoint.
[2025-02-15 17:15:06] [INFO] /
[2025-02-15 17:15:06] [INFO] FLUXモデルをfp8に変換し
[2025-02-15 17:15:06] [INFO] ています。これには時間が
[2025-02-15 17:15:06] [INFO] かかる場合があります。fp
[2025-02-15 17:15:06] [INFO] 8チェックポイントを使用
[2025-02-15 17:15:06] [INFO] することで時間を短縮でき
[2025-02-15 17:15:06] [INFO] ます。
[2025-02-15 17:17:28] [INFO] 2025-02-15 17:17:28 INFO     enable block swap:       flux_train_network.py:115
[2025-02-15 17:17:28] [INFO] blocks_to_swap=18
[2025-02-15 17:17:29] [INFO] FLUX: Block swap enabled. Swapping 18 blocks, double blocks: 9, single blocks: 18.
[2025-02-15 17:17:29] [INFO] 2025-02-15 17:17:29 INFO     Building CLIP-L                  flux_utils.py:179
[2025-02-15 17:17:29] [INFO] INFO     Loading state dict from          flux_utils.py:275
[2025-02-15 17:17:29] [INFO] E:\ai\github\fluxgym\models\clip
[2025-02-15 17:17:29] [INFO] \clip_l.safetensors
[2025-02-15 17:17:31] [INFO] 2025-02-15 17:17:31 INFO     Loaded CLIP-L: <All keys matched flux_utils.py:278
[2025-02-15 17:17:31] [INFO] successfully>
[2025-02-15 17:17:32] [INFO] 2025-02-15 17:17:32 INFO     Loading state dict from          flux_utils.py:330
[2025-02-15 17:17:32] [INFO] E:\ai\github\fluxgym\models\clip
[2025-02-15 17:17:32] [INFO] \t5xxl_fp16.safetensors
[2025-02-15 17:17:32] [INFO] INFO     Loaded T5xxl: <All keys matched  flux_utils.py:333
[2025-02-15 17:17:32] [INFO] successfully>
[2025-02-15 17:17:32] [INFO] INFO     Building AutoEncoder             flux_utils.py:144
[2025-02-15 17:17:32] [INFO] INFO     Loading state dict from          flux_utils.py:149
[2025-02-15 17:17:32] [INFO] E:\ai\github\fluxgym\models\vae\
[2025-02-15 17:17:32] [INFO] ae.sft
[2025-02-15 17:17:33] [INFO] 2025-02-15 17:17:33 INFO     Loaded AE: <All keys matched     flux_utils.py:152
[2025-02-15 17:17:33] [INFO] successfully>
[2025-02-15 17:17:33] [INFO] import network module: networks.lora_flux
[2025-02-15 17:17:33] [INFO] INFO     [Dataset 0]                     train_util.py:2589
[2025-02-15 17:17:33] [INFO] INFO     caching latents with caching    train_util.py:1097
[2025-02-15 17:17:33] [INFO] strategy.
[2025-02-15 17:17:33] [INFO] INFO     caching latents...              train_util.py:1146

**Comment by Latentnaut** [comment]
I'm in the same page

**Comment by DamienCz** [comment]
same

**Comment by RomanticQq** [comment]
me too

**Comment by jimmaybones** [comment]
Was working fine earlier, now stuck at "caching latents..."
